jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735BrakeSystemStatus.java

```diff
@@ -20,18 +20,18 @@
 public class J2735BrakeSystemStatus extends Asn1Object {
     private static final long serialVersionUID = 1L;
 
-    private J2735BrakeAppliedStatus wheelBrakes;
+    private J2735BitString wheelBrakes;
```

Comment:
Let's put the definition of BrakeAppliedStatus enum and remove it from BitStringBuilder.